### DISCUSSION TITLE
MPC Player Black Screen

### DISCUSSION THREAD

**Discussion by trufflesq** [streamlink/streamlink#3925] [Q&A] [answered]
Whenever I run another application/game that has like an admin prompt or ctrl + L windows, my MPC player (that has a twitch stream open), alt tabbing from a fullscreen app/game the player pauses or turns into a black screen, but it continues playing after the app/game has been launched.  Any idea on how to fix this ? Thanks 

Note: The problem doesn't happen when I use VLC but I mostly prefer to use MPC player over it.

**Comment by bastimeyer** [comment, marked as answer]
You'll have to ask on the forums / bug trackers of the specific MPC version you're using (the main MPC project is dead since 2017). Your issue is unrelated to Streamlink and can't be solved here.
https://en.wikipedia.org/wiki/Media_Player_Classic

**Reply by trufflesq** [reply]
Okay, thank you
